tests/testthat/test_stan_clogit.R

```diff
@@ -0,0 +1,69 @@
+# Part of the rstanarm package for estimating model parameters
+# Copyright (C) 2017 Trustees of Columbia University
+# 
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# tests can be run using devtools::test() or manually by loading testthat 
+# package and then running the code below possibly with options(mc.cores = 4).
+
+# this mostly goes through same code as a logit model so only testing the unique stuff
+
+library(rstanarm)
+
+SEED <- 123
+ITER <- 100
+CHAINS <- 2
+CORES <- 1
+REFRESH <- 0
+
+threshold <- 0.01
+
+source(file.path("helpers", "expect_stanreg.R"))
+
+context("stan_clogit")
+
+fit <- stan_clogit(case ~ spontaneous + induced, strata = stratum, prior = NULL,
```

Comment:
Probably should test the multilevel syntax also. If we don't have a comparison for that, we can at least test that it runs.